feat(Necord): add presence

SocketSomeone opened the pull request:
## Description 
<!-- A clear and detailed description of the changes, referencing issues if applicable -->

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

![image](https://user-images.githubusercontent.com/30754373/222884751-5f306c37-68d8-4f55-b12e-be5d9fa4b840.png)

![image](https://user-images.githubusercontent.com/30754373/222884800-664f216d-53cc-43d4-be88-658c18d7d4b9.png)

![image](https://user-images.githubusercontent.com/30754373/222884815-31d6b73e-f7e7-42af-a68a-882ab05c7338.png)



</details>

RisingSunLight42 reviewed (changes requested):
Really low quality presence, try to add more features on it or It will be refused

SocketSomeone commented:
What mean "add more features"? I added all features of necord documentation site.

Similar  wiki website was merged https://github.com/PreMiD/Presences/pull/6213
(Used as example)

I just removed blog cuz @necordjs doesnt have any blogs at website and small images

RisingSunLight42 commented:
> @RisingSunLight42 @EncryptedDev Hi there,
> 
> What mean "add more features"? I added all features of necord documentation site.
> 
> Similar wiki website was merged #6213 (if that got merged, then i don't see any issue with this pull request) (Used as example)
> 
> I just removed blog and small images cuz @necordjs doesnt have any blogs at website
> 
> I would be glad if you could tell me what features need to be added so that this is merged
> 
> Thank you!

Between the PR #6213 and this one, a lot of time as passed (almost one year).
The PR you've mentionned would not pass the validation now, PreMiD ask now for more features on the presence and not something static.
I don't know Necordjs, but I'm sure some things can be done and imagined to enhance the presence to make it not just be a "I'm on this page".
